src/arrayMethodFilter.js

```diff
@@ -5,7 +5,12 @@
  */
 function applyCustomFilter() {
   [].__proto__.filter2 = function(callback) {
-    // write code here
+    let result = [];
+    this.map((el, i) => {
```

Comment:
avoid using `map` when you need only to process the array without returning any value from the loop